templates/monorepo/lerna.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "0.0.0",
+  "registry": "https://registry.npmjs.org/",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
are these necessary? I believe these are the defaults

Reply:
not for scoped packages